### PR TITLE
Added Parallel Jim Investigator

### PR DESCRIPTION
**Pull request by Entrox-Licher**
Added Parallel Jim Investigator (not his challenge scenario yet) and some minor fixes to old cards.

### CODE REVIEW

**Comment by Chr1Z93** [comment]
You need to update the id ranges here too:
https://github.com/argonui/SCED/blob/main/src/arkhamdb/ArkhamDb.ttslua#L200C6-L216C10

**Comment by Entrox-Licher** [comment]
Done!  We should probably refactor that later to be some constant somewhere so it's easier to catch

File: objects/AllPlayerCards.15bb07/JimsTrumpet.03c6a7.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
       "UniqueBack": false
     }
   },
-  "Description": "",
+  "Description": "The Dead Speak",
```
Comment (Chr1Z93) [inline objects/AllPlayerCards.15bb07/JimsTrumpet.03c6a7.json:25]: This should be "The Dead Listen"

File: objects/AllPlayerCards.15bb07/JimsTrumpet.7dfd5f.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      "BackIsHidden": false,
+      "BackURL": "https://i.imgur.com/EcbhVuh.jpg/",
+      "FaceURL": "http://cloud-3.steamusercontent.com/ugc/2149964195986880793/517FBB4FF8F72900B9E123DB865BCAD625F6506C/",
+      "NumHeight": 2,
+      "NumWidth": 2,
+      "Type": 0,
+      "UniqueBack": false
+    }
+  },
+  "Description": "The Dead Speak (Advanced)",
```
Comment (Chr1Z93) [inline objects/AllPlayerCards.15bb07/JimsTrumpet.7dfd5f.json:25]: This should be "The Dead Speak"

File: src/arkhamdb/DeckImporterMain.ttslua [hunk 3]
```diff
@@ -322,6 +323,48 @@ function handleHunchDeck(investigatorId, cardList, playerColor)
...
+  if investigatorId == "02004-p" or investigatorId == "02004-pb" then -- Parallel Jim Culver
+    local spritList = {}
+    for i, card in ipairs(cardList) do
+      if card.metadata.id == "90053" or (
+          card.metadata.type == "Asset"
+          and card.metadata.traits ~= nil
+          and string.match(card.metadata.traits, "Ally")
+          and card.metadata.level ~= nil
+          and card.metadata.level < 3) then
+        table.insert(spritList, i)
```
Comment (Chr1Z93) [inline src/arkhamdb/DeckImporterMain.ttslua:342]: let's name this "spiritList" instead of "spritList" 😉

**Review by Chr1Z93 — COMMENTED** [review]
Some nitpicking

File: objects/AllPlayerCards.15bb07/JimsTrumpet.7dfd5f.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
       "UniqueBack": false
     }
   },
-  "Description": "The Dead Speak (Advanced)",
+  "Description": "The Dead Listen (Advanced)",
```
Comment (Chr1Z93) [inline objects/AllPlayerCards.15bb07/JimsTrumpet.7dfd5f.json:25]: The advanced trumpet does actually say "The Dead Speak" 😅

**Review by Chr1Z93 — CHANGES_REQUESTED** [review]

**Review by Chr1Z93 — APPROVED** [review]

**Pull request merged**